src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -184,7 +184,7 @@ define([
             // getter for content data
             this.sandbox.on('sulu.content.contents.get-data', function(callback) {
                 // deep copy of object
-                callback(JSON.parse(JSON.stringify(this.data)));
+                callback(this.sandbox.util.deepCopy(this.data));
```

Comment:
i think its a clear direct dependency ... but ok (: we could discuss the service concept afterwards (: